src/components/PartyAccountItemButton/PartyAccountItemButton.tsx

```diff
@@ -127,7 +128,7 @@ export const PartyAccountItemButton = ({
                 component="h6"
                 color="inherit"
                 sx={{
-                  fontWeight: theme.typography.fontWeightBold,
+                  fontWeight: "600",
```

Comment:
Isn't there a way to extend `typography` to accept a `fontWeightSemibold` key?
```suggestion
                  fontWeight: theme.typography.fontWeightSemibold,
```

Reply:
I now see that in the theme.ts file  already exist theme.typography.fontWeightMedium with the value of 600 so i can use that.
Would you like i change the name from fontWeightMedium => fontWeightSemibold ?